application/src/Site/BlockLayout/BrowsePreview.php

```diff
@@ -23,6 +23,7 @@ public function form(PhpRenderer $view, SiteRepresentation $site,
             'query' => '',
             'heading' => '',
             'limit' => 12,
+            'components' => ["resource-heading","resource-body","thumbnail"],
```

Comment:
Just simple style matters: simple strings should be using single quotes, and there should be a space after every comma. Same applies to this same array appearing in `render`.